home/pages/home.css

```diff
@@ -1,5 +1,5 @@
 @import url('../vendor/normalize.css');
-@import url('../vendor/fonts.css');
+@import url('https://fonts.googleapis.com/css2?family=Heebo:wght@400;500;700;900&family=Inter:wght@400;600;900&display=swap');
```

Comment:
did exactly the same thing parallel to you and yet I still think that the 900 font doesn't work. Please checkout my PR as well